pom.xml

```diff
@@ -543,6 +537,12 @@
             <version>1.1.10.1</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.22</version>
```

Comment:
Why do we need to roll back the version from `1.18.24` to `1.18.22`?

Reply:
IntelliJ did that

Reply:
You dont